src/main/java/jep/Jep.java

```diff
@@ -605,384 +396,41 @@ private native void set(long tstate, String name, Object v)
     private native void set(long tstate, String name, Class<?> v)
             throws JepException;
 
-    /**
-     * Sets the Java String into the sub-interpreter's global scope with the
-     * specified variable name.
-     * 
-     * @deprecated Use {@link #set(String, Object)} instead.
-     * 
-     * @param name
-     *            the Python name for the variable
-     * @param v
-     *            a <code>String</code> value
-     * @throws JepException
-     *             if an error occurs
-     */
-    @Deprecated
-    public void set(String name, String v) throws JepException {
-        isValidThread();
-
-        set(tstate, name, v);
-    }
-
     private native void set(long tstate, String name, String v)
             throws JepException;
 
-    /**
-     * Sets the Java boolean into the sub-interpreter's global scope with the
-     * specified variable name.
-     * 
-     * @deprecated Use {@link #set(String, Object)} instead.
-     * 
-     * @param name
-     *            the Python name for the variable
-     * @param v
-     *            a <code>boolean</code> value
-     * @throws JepException
-     *             if an error occurs
-     */
-    @Deprecated
-    public void set(String name, boolean v) throws JepException {
-        isValidThread();
-
-        set(tstate, name, v);
-    }
-
-    private native void set(long tstate, String name, boolean v)
```

Comment:
When you delete native methods you should also delete the implementation in jep.c

Reply:
That was removed by accident, I added it again.